index.html

```diff
@@ -208,6 +213,7 @@ <h1>2020 NY Democratic Primary Election Results</h1>
 var info_columns = ["ad_ed", "reg_dems", "absentee_requested", "total"],
     table_cols = [
       {head: "Candidate", class: "name-cell", html: r => r.candidate }, 
+      {head: "", class: "name-cell", html: r => r.candidate},
```

Comment:
Also couldn't quite figure out what was going on with this array, so I added sort of a dummy entry and it worked 🤷